test/tests.yml

```diff
@@ -127,6 +162,16 @@
     - name: azure_rockylinux
     - name: gce_default
 
+- scenario: install_containerd
+  initVersion: v1.29.2
+  infrastructures:
+    - name: azure_centos
+    - name: azure_default
+    - name: azure_flatcar
+    - name: azure_rhel
+    - name: azure_rockylinux
+    - name: gce_default
```

Comment:
The `install_containerd` scenario is based on the in-tree CCM and Kubernetes doesn't support in-tree CCMs as of 1.29. That said, this scenario/version combination is not possible at all (at least not in KubeOne).